### PR TITLE
css

### PR DESCRIPTION
**Pull request by Hunter-aw**
I did lots of stuff

### CODE REVIEW

File: dist/style.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+body{
+    background-color: blueviolet;
+    opacity: 0.1
+}
+
+div{
+    font-size: 3em;
+    color: blanchedalmond;
+}
```
Comment (Hunter-aw) [inline dist/style.css:9]: this is a bad color, shame on you

**Review by pagella42 — COMMENTED** [review]
you should get fired

**Comment by pagella42** [comment]
> you should get fired

bad code

File: dist/style.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+body{
+    background-color: blueviolet;
+    opacity: 0.1
```
Comment (Axel936) [inline dist/style.css:3]: This is amazing! good job!

**Comment by neb-sidnal** [comment]
blanched - (of an item of food) having been briefly immersed in boiling water, especially in order to remove the skin or to prepare it for further cooking.

**Pull request merged**